doc/radosgw/multisite.rst

```diff
@@ -400,7 +400,7 @@ section of the guide. The configuration section for radosgw should resemble::
 
   [client.rgw.us-east-1]
   rgw_frontends="civetweb port=80"
-  rgw_zone=us-east
+  rgw_zone=us-east-1
```

Comment:
The previous doc mentioned only three zones:

```
 83 - Master zone: United States, East Region 1: ``us-east-1``
 84 - Secondary zone: United States, East Region 2: ``us-east-2``
 85 - Secondary zone: United States, West Region: ``us-west``
```
so the zone name should be `us-east-1` ?